catroid/src/main/java/org/catrobat/catroid/content/admob/AdMobBanner.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2020 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.content.admob
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.View
+import android.widget.RelativeLayout
+import com.google.android.gms.ads.AdRequest
+import com.google.android.gms.ads.AdSize
+import com.google.android.gms.ads.AdView
+import com.google.android.gms.ads.MobileAds
+import com.google.android.gms.ads.RequestConfiguration
+import com.google.android.gms.ads.RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE
+import org.catrobat.catroid.BuildConfig
+
+class AdMobBanner
+@SuppressLint("HardwareIds")
+constructor(context: Context) {
+
+    val adView: AdView = AdView(context)
+    // private val deviceId: String = Utils.getAdMobDeviceId(context)
+
+    fun createNew(adSize: AdSize?, relativeLayoutRule: Int) {
+        adView.adSize = adSize
+        adView.adUnitId = BuildConfig.ADMOB_BANNER_UNIT_ID
+        // val testDeviceIds = listOf(deviceId)
+        val configuration = RequestConfiguration.Builder()
+            .setTagForChildDirectedTreatment(TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE)
+            // .setTestDeviceIds(testDeviceIds)
```

Comment:
are the comments intended

Reply:
Please also remove the other comments (https://confluence.catrob.at/display/KNOWHOW/Clean+code)